katago_server/games/serializers/rating_game.py

```diff
@@ -47,6 +47,10 @@ class Meta:
             "black_network": {"lookup_field": "name"},
         }
 
+    def validate(self,data):
+        if not data["white_network"].rating_games_enabled or not data["black_network"].rating_games_enabled:
```

Comment:
In which case do we want to disable an network for rating games ?

Reply:
Maybe someone uploads some experimental net that is not part of the main run that we would like to test, and then we get data on it, and then disable.

Maybe we decide that all weaker and older nets no longer benefit from testing since they are so old and nobody cares about them, but they still sometimes come up because after enough rating games, newer networks become more certain than they are so the older networks then become "most uncertain". So we could disable them.

Etc.  There may be many reasons.